validate-date.js

```diff
@@ -1,31 +1,50 @@
-function validateDate(dateValue) {
-    if (dateValue == null) {
-        return 'Invalid Format';
-    }
-    return dateValidator(dateValue);
+function validateDate(dateValue, boolean = false) {
+  let responses = boolean
+    ? [false, false, false, true]
+    : ["Invalid Format", "Invalid Date", "Invalid Format", "Valid Date"];
```

Comment:
just noticing.. can we get rid of either of the "Invalid Format" in the response. Curious why you added it two times. :)

Reply:
definitely! I just got all the returns from the original and added to this ternary...didn't realize two of those were the same